mockup-contributions_list.php

```diff
@@ -0,0 +1,922 @@
+<?php
```

Comment:
A good thing is, if everything is in one file, you can approve those pull requests / merges easier as well, because there is only one file to check. and everything is scoped. I would say lets add some additional comments to the templates soon, expressing that both js and css should be inlined in the file, and it is temporary.